opus/application/apps/ui/views.py

```diff
@@ -636,7 +636,9 @@ def api_init_detail_page(request, **kwargs):
         products = {}
     new_products = OrderedDict()
     for version in products:
-        new_products[version] = OrderedDict()
+        # Display "Version: 1.0" instead of "Version: 1" in detail tab
+        pretty_ver = version if version == 'Current' else f'{float(version)}'
```

Comment:
Instead of using the f'' notation it's more clear just to say str(float(version)) unless you're planning to actually use the f'' formatting in some way.